wire-ios-request-strategy/Sources/Request Strategies/Assets/AssetV3DownloadRequestStrategyTests.swift

```diff
@@ -382,10 +385,10 @@ extension AssetV3DownloadRequestStrategyTests {
         }
     }
 
-//        When the backend redirects to the cloud service to get the image, it could be that the
-//        network bandwidth of the device is really bad. If the time interval is pretty long before
-//        the connectivity returns, the cloud responds with an error having status code 403
-//        -> retry the image request and do not delete the asset client message.
+    //        When the backend redirects to the cloud service to get the image, it could be that the
+    //        network bandwidth of the device is really bad. If the time interval is pretty long before
+    //        the connectivity returns, the cloud responds with an error having status code 403
+    //        -> retry the image request and do not delete the asset client message.
```

Comment:
```suggestion
    // When the backend redirects to the cloud service to get the image, it could be that the
    // network bandwidth of the device is really bad. If the time interval is pretty long before
    // the connectivity returns, the cloud responds with an error having status code 403
    // -> retry the image request and do not delete the asset client message.
```